src/mqt/core/_core/__init__.pyi

```diff
@@ -1 +1,733 @@
+    def is_unitary(self) -> bool: ...
+    @property
+    def n_targets(self) -> int: ...
+    @property
+    def n_qubits(self) -> int: ...
+    @property
+    def n_controls(self) -> int: ...
+    @property
+    def name(self) -> str: ...
+    @name.setter
+    def name(self, arg: str) -> None: ...
+    def set_n_qubits(self, arg: int, /) -> None: ...
+    def size(self) -> int: ...
+    @property
+    def targets(self) -> list[int]: ...
+    @targets.setter
+    def targets(self, arg: list[int]) -> None: ...
+    @property
+    def gate(self) -> OpType: ...
+    @gate.setter
+    def gate(self, arg: OpType) -> None: ...
+    def to_open_qasm(self, qreg: list[tuple[str, str]], creg: list[tuple[str, str]], /) -> str: ...
+
+class NonUnitaryOperation(Operation):
+    @overload
+    def __init__(self, arg0: int, arg1: list[Operation], /) -> None: ...
+    @overload
+    def __init__(self, nq: int, qubit_register: list[int], classical_register: list[int], /) -> None: ...
+    @overload
+    def __init__(self, nq: int, qubit: int, bit: int, /) -> None: ...
+    @overload
+    def __init__(self, nq: int, qubit_register: list[int], n: int) -> None: ...
+    @overload
+    def __init__(self, nq: int) -> None: ...
+    @overload
+    def __init__(self, nq: int, qubit_register: list[int], gate: OpType) -> None: ...
+    def acts_on(self, arg: int, /) -> bool: ...
+    def add_depth_contribution(self, arg: list[int], /) -> None: ...
+    def clone(self) -> Operation: ...
+    @property
+    def controls(self) -> set[Control]: ...
+    @controls.setter
+    def controls(self, arg: set[Control]) -> None: ...
+    def empty(self) -> bool: ...
+    def equals(self, arg0: Operation, arg1: Permutation, arg2: Permutation, /) -> bool: ...
+    def is_classic_controlled_operation(self) -> bool: ...
+    def is_compound_operation(self) -> bool: ...
+    def is_controlled(self) -> bool: ...
+    def is_non_unitary_operation(self) -> bool: ...
+    def is_standard_operation(self) -> bool: ...
+    def is_symbolic_operation(self) -> bool: ...
+    def is_unitary(self) -> bool: ...
+    @property
+    def n_targets(self) -> int: ...
+    @property
+    def n_qubits(self) -> int: ...
+    @property
+    def n_controls(self) -> int: ...
+    @property
+    def name(self) -> str: ...
+    @name.setter
+    def name(self, arg: str) -> None: ...
+    def size(self) -> int: ...
+    @property
+    def targets(self) -> list[int]: ...
+    @targets.setter
+    def targets(self, arg: list[int]) -> None: ...
+    @property
+    def gate(self) -> OpType: ...
+    @gate.setter
+    def gate(self, arg: OpType) -> None: ...
+    @property
+    def classics(self) -> list[int]: ...
+    def to_open_qasm(self, qreg: list[tuple[str, str]], creg: list[tuple[str, str]], /) -> str: ...
+
+class StandardOperation(Operation):
+    @overload
+    def __init__(
+        self,
+        nq: int,
+        controls: set[Control],
+        target0: int,
+        target1: int,
```

Comment:
This is surely missing the default argument. I haven't checked all functions right now, but please make sure that every function that has default arguments also has them in the stubs.

Reply:
Should all be in there now